packages/calcite-design-tokens/src/brand/global.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What is the reason for committing these files containing empty JSON objects? Is content supposed to be added to them eventually?

Reply:
yes this is a placeholder that needs to stay for now.